.travis.yml

```diff
@@ -25,20 +25,17 @@ env:
 matrix:
   fast_finish: true
   include:
-  - env: LINT="dev"
+  - env: CLIPPY='yes'
     rust: nightly
   allow_failures:
-  - env: LINT="dev"
+  - env: CLIPPY='yes'
 
 script:
   # Test common features.
   - cargo test --features="$FEATURES"
   - |
-    if [ $TRAVIS_RUST_VERSION = 'nightly' ]; then
-        # Enable dev for clippy linting.
-        cargo test --features="$FEATURES $LINT";
-        # Lint nightly feature.
-        cargo test --features="$FEATURES $LINT nightly";
+    if [ $CLIPPY = 'yes' ]; then
```

Comment:
Since CIPPY is always `yes`, can we remove this condition test?

Reply:
the CLIPPY env is only  set yes when rust is nightly on the matrix section,  so I thought the condition test is still need

Reply:
Oh. Make sense.